test/cppsuite/test_harness/configuration_settings.h

```diff
@@ -94,9 +99,9 @@ class configuration {
     }
 
     int
-    get(const char *key, WT_CONFIG_ITEM *value)
+    get(const std::string &key, WT_CONFIG_ITEM *value)
     {
-        return (_config_parser->get(_config_parser, key, value));
+        return (_config_parser->get(_config_parser, key.c_str(), value));
     }
 
     private:
```

Comment:
I feel these fields can also be const. Is there a need to modify them?

Reply:
_config_parser is set in the constructor, see lines 19 and 33